templates/docs/examples/patterns/grid/_cve-table.html

```diff
@@ -0,0 +1,539 @@
+<table class="cve-table">
+  <thead>
+    <tr>
+      <th style="width: 10em">ID</th>
+      <th>Priority</th>
+      <th>Package</th>
+      <th>
+        <div class="icon-container__text"> 14.04 ESM </div>
```

Comment:
I guess that these class names `icon-container__text` `cve-table-cell-priority`, etc are just copy paste from the CVE page, but served as part of Vanilla example they seem a bit confusing as they don't provide any styling so there is no use for them in the example.

Reply:
I've cleaned up the entire file